### PR TITLE
Use Apache instead of Nginx with Docker

### PR DESCRIPTION
**Pull request by caitlin**
We should have our development environment replicate our production environment as closely as possible. Since that is what we use today for the production server, this update changes our docker configuration to use apache instead of nginx.

If we decide to switch to nginx, we can change the docker configuration at that time.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -24,6 +17,8 @@ services:
       - ./werewolf:/var/www/html
       - ./php/php.conf:/usr/local/etc/php-fpm.d/php.conf
       - ./php/php.ini:/usr/local/etc/php/php.ini
```
Comment (PilotBob) [inline docker-compose.yml:19]: Are these still the correct locations? I would expect them to be different than the php-fpm image.

**Review by PilotBob — COMMENTED** [review]
I pulled down the container. The path php-fpm isn't in that container at all. Per that docs you should use  $PHP_INI_DIR for the ini file and $PHP_INI_DIR/conf.d for the config file. 

I did start researching if there could be more than one ini or config file.. because they suggest you use php.ini-production or php.ini-development depending on what you are doing. But, didn't get too far.

https://hub.docker.com/_/php

File: docker-compose.yml (hunk 1 above)
Comment (caitlin) [reply docker-compose.yml:19]: Probably not.

**Comment by caitlin** [comment]
How do we get php-fpm and apache together? I had trouble figuring out how to do that.

**Comment by PilotBob** [comment]
you don't. See discourse reply.

**Pull request closed without merging**